### PR TITLE
docs: translations for components progress

### LINKED ISSUE #426: `<progress>`
Page: https://id.react.dev/reference/react-dom/components/progress

Assignee: @ilham89

### PR DESCRIPTION
**Pull request by ilham89**
Hi, this is a Pull Request for https://github.com/reactjs/id.react.dev/issues/330
Translation for page https://react.dev/reference/react-dom/components/progress

Closes #426

### CODE REVIEW

File: src/content/reference/react-dom/components/progress.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ title: "<progress>"
 
 <Intro>
 
-The [built-in browser `<progress>` component](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/progress) lets you render a progress indicator.
+[Komponen `<progress>` bawaan browser](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/progress) memungkinkan Anda untuk menampilkan indikator progres.
```
Comment (mazipan) [inline src/content/reference/react-dom/components/progress.md:7]:
```suggestion
[Komponen `<progress>` bawaan peramban](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/progress) memungkinkan Anda untuk menampilkan indikator progres.
```

File: src/content/reference/react-dom/components/progress.md [hunk 2]
```diff
@@ -16,36 +16,36 @@ The [built-in browser `<progress>` component](https://developer.mozilla.org/en-U
 
 ---
 
-## Reference {/*reference*/}
+## Referensi {/*reference*/}
 
 ### `<progress>` {/*progress*/}
 
-To display a progress indicator, render the [built-in browser `<progress>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/progress) component.
+Untuk menampilkan indikator progres, render komponen [`<progress>` bawaan browser](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/progress).
```
Comment (mazipan) [inline src/content/reference/react-dom/components/progress.md:23]:
```suggestion
Untuk menampilkan indikator progres, render komponen [`<progress>` bawaan peramban](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/progress).
```

File: src/content/reference/react-dom/components/progress.md [hunk 3]
```diff
@@ -16,36 +16,36 @@ The [built-in browser `<progress>` component](https://developer.mozilla.org/en-U
...
 
-`<progress>` supports all [common element props.](/reference/react-dom/components/common#props)
+`<progress>` mendukung semua [elemen umum *props*.](/reference/react-dom/components/common#props)
 
-Additionally, `<progress>` supports these props:
+Selain itu, `<progress>` juga mendukung *props*:
 
-* [`max`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/progress#attr-max): A number. Specifies the maximum `value`. Defaults to `1`.
-* [`value`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/progress#attr-value): A number between `0` and `max`, or `null` for intermedinate progress. Specifies how much was done.
+* [`max`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/progress#attr-max): Sebuah angka. Menentukan nilai maksimum `value`. Default ke `1`.
```
Comment (mazipan) [inline src/content/reference/react-dom/components/progress.md:37]: Kalau mau pakai Default, pakai italic.
Kalau mau pakai Bahasa Indonesia, pakai `Bawaan`

**Comment by r17x** [comment]
Any updates @ilham89 ?

**Comment by ilham89** [comment]
> Any updates @ilham89 ?

sorry since yesterday I had a fever. thanks for the input and reminder. I will update tonight

File: src/content/reference/react-dom/components/progress.md (hunk 1 above)
Comment (ilham89) [reply src/content/reference/react-dom/components/progress.md]: udah aku update mas, bisa dicek minta tolong lagi

File: src/content/reference/react-dom/components/progress.md (hunk 2 above)
Comment (ilham89) [reply src/content/reference/react-dom/components/progress.md]: udah aku update mas, bisa dicek minta tolong lagi

File: src/content/reference/react-dom/components/progress.md (hunk 3 above)
Comment (ilham89) [reply src/content/reference/react-dom/components/progress.md]: udah aku update mas, bisa dicek minta tolong lagi

File: src/content/reference/react-dom/components/progress.md [hunk 4]
```diff
@@ -16,36 +16,36 @@ The [built-in browser `<progress>` component](https://developer.mozilla.org/en-U
...
 
-To display a progress indicator, render the [built-in browser `<progress>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/progress) component.
+Untuk menampilkan indikator progres, render komponen [`<progress>` bawaan peramban](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/progress).
 
 ```js
 <progress value={0.5} />
 ```
 
-[See more examples below.](#usage)
+[Lihat contoh-contoh lainnya di bawah ini.](#usage)
```
Comment (mazipan) [inline src/content/reference/react-dom/components/progress.md:29]:
```suggestion
[Lihat lebih banyak contoh lainnya di bawah.](#usage)
```

File: src/content/reference/react-dom/components/progress.md [hunk 5]
```diff
@@ -16,36 +16,36 @@ The [built-in browser `<progress>` component](https://developer.mozilla.org/en-U
...
+## Penggunaan {/*usage*/}
 
-### Controlling a progress indicator {/*controlling-a-progress-indicator*/}
+### Mengontrol indikator progres {/*controlling-a-progress-indicator*/}
 
-To display a progress indicator, render a `<progress>` component. You can pass a number `value` between `0` and the `max` value you specify. If you don't pass a `max` value, it will assumed to be `1` by default.
+Untuk menampilkan indikator progres, render komponen `<progress>`. Anda dapat memberikan nilai `value` antara `0` dan nilai `max` yang anda tentukan. Jika Anda tidak memberikan nilai `max`, nilai tersebut akan dianggap sebagai `1` secara *default*.
 
-If the operation is not ongoing, pass `value={null}` to put the progress indicator into an indeterminate state.
+Jika operasi tidak sedang berlangsung, berikan `value={null}` untuk mendapatkan indikator progres ke status tidak tentu.
```
Comment (mazipan) [inline src/content/reference/react-dom/components/progress.md:48]:
```suggestion
Jika operasi tidak sedang berlangsung, berikan `value={null}` untuk mendapatkan indikator progres ke *state* tidak tentu (*indeterminate*).
```

File: src/content/reference/react-dom/components/progress.md [hunk 6]
```diff
@@ -16,36 +16,36 @@ The [built-in browser `<progress>` component](https://developer.mozilla.org/en-U
...
 ---
 
-## Usage {/*usage*/}
+## Penggunaan {/*usage*/}
 
-### Controlling a progress indicator {/*controlling-a-progress-indicator*/}
+### Mengontrol indikator progres {/*controlling-a-progress-indicator*/}
 
-To display a progress indicator, render a `<progress>` component. You can pass a number `value` between `0` and the `max` value you specify. If you don't pass a `max` value, it will assumed to be `1` by default.
+Untuk menampilkan indikator progres, render komponen `<progress>`. Anda dapat memberikan nilai `value` antara `0` dan nilai `max` yang anda tentukan. Jika Anda tidak memberikan nilai `max`, nilai tersebut akan dianggap sebagai `1` secara *default*.
```
Comment (mazipan) [inline src/content/reference/react-dom/components/progress.md:46]:
```suggestion
Untuk menampilkan indikator progres, render komponen `<progress>`. Anda dapat mengoper nilai `value` antara `0` dan nilai `max` yang anda tentukan. Jika Anda tidak memberikan nilai `max`, nilai tersebut akan dianggap sebagai `1` secara *default*.
```

File: src/content/reference/react-dom/components/progress.md (hunk 4 above)
Comment (ilham89) [reply src/content/reference/react-dom/components/progress.md]: I've updated it, please help to review again. thank you

File: src/content/reference/react-dom/components/progress.md (hunk 5 above)
Comment (ilham89) [reply src/content/reference/react-dom/components/progress.md]: I've updated it, please help to review again. thank you

File: src/content/reference/react-dom/components/progress.md (hunk 6 above)
Comment (ilham89) [reply src/content/reference/react-dom/components/progress.md]: I've updated it, please help to review again. thank you

**Review by mazipan — APPROVED** [review]

**Review by r17x — APPROVED** [review]

**Comment by r17x** [comment]
Thanks for contribution 🥳  @ilham89 @mazipan @mhaidarhanif

**Pull request merged**